File: scripts/generate_demo_gif.py
#!/usr/bin/env python3
"""Legacy redirect to 100% authentic demo GIF generator."""
import os
import sys

target = os.path.join(os.path.dirname(__file__), "generate_real_demo_gifs.py")
os.execv(sys.executable, [sys.executable, target] + sys.argv[1:])\n

File: scripts/generate_demo_gif.js
#!/usr/bin/env node
/** Legacy redirect to 100% authentic demo GIF generator. */
'use strict';

const path = require('path');
const { spawn } = require('child_process');

const target = path.join(__dirname, 'generate_real_demo_gifs.js');
const child = spawn(process.execPath, [target, ...process.argv.slice(2)], {
  stdio: 'inherit',
});

child.on('exit', (code, signal) => {
  if (signal) {
    process.kill(process.pid, signal);
  } else {
    process.exit(code == null ? 1 : code);
  }
});
